charts/calibre-web/1.2.0/questions.yaml

```diff
@@ -1,47 +1,52 @@
+# Groups
```

Comment:
why did you modify calibre?

Reply:
It's a shame, I could've merged it right now, but you choose to commit bad changes into it.
Now you NEED to remove this chance from the log. Which means rebasing and squashing everything.

Don't just create a new branch, get a good UI tool and learn to do it.

Reply:
Ok fixed calibre, i don't how i did that, i'll see if i can rebase and squash

Reply:
@stavros-k It doesn't just need to be removed, it needs to be removed from the history, thats why a squash and/or a rebase is needed.

Reply:
I see, it looks to be gone from history now...
Anyway, try and get a good tool to squash and rebase, because you'll need it more often.

Reply:
I'll look into it tomorrow, i'm afraid to try any squash on this repo, all messages says that "this cant be undone", and dont want to mess anything again.
I'll make a new repo to do some excercise there first. 
If you want you can fix this one so it can be merged, or i'll do it in 1-2 days when i'm feeling comfortable doing this.

Reply:
@stavros-k ofcoarse it can be undone, just create a new branch beforehand as a backup.
Also: Stop using your master branch for PR's, it's bad practice.

Reply:
Also: Stop merging (pulling) the master branch into this one. 
If it creates rather unclean history, with extra (merge) commits everywhere.

If you want to update a branch, you can rebase it.
(I'm not sure why github even added the button to "update" a PR, because its not best practice to use it anyway)

Reply:
Yes, i see now the reason.

Reply:
Sorry with all this merges i made, i made a mess and  i can't figure out how to squash.

Reply:
Use Source tree.
Right click the latest commit before yours, select "interactive rebase", select top commit in the new window, hit "squash with previous" till all your own commits are squashed and click okey.

Reply:
When done, you need to force push, not push.

Reply:
It just reports a ton of conflicts, when i close the window, hoping it will ask me how to resolve them, it just undo everything.
😒

Reply:
ahh, yeah...  
In that case:
- copy the App folder to a folder outside the repo.
- rightclick the last master commit (the one before your earliest commit) and "reset to this commit"
- then force push
- delete the current app folder (if present, in this case it's not present so skip this)
- copy back the app folder you moved away.
- push
- profit.

Yes it's not "good git practice", but with multiple squash conflicts. With the above method the end result is like 99% the same as squashing everything. Without opening multiple PR's annoying the crap out of every maintainer.

Reply:
Hmm , this closed the PR. Should a open a new?